doc/zmq_setsockopt.txt

```diff
@@ -87,6 +87,21 @@ Default value:: not set
 Applicable socket types:: all, when using TCP or UDP transports.
 
 
+ZMQ_BUSY_POLL: This removes delays caused by the interrupt and the resultant context switch.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
The `~` should match in length the line above, otherwise the formatting in the manpage will be wrong